source/documentation/data-docs/curated-databases-docs/databases-for-analysis-and-apps.md

```diff
@@ -0,0 +1,58 @@
+
+If in doubt about best practices we have slack channels for R, python, SQL, etc where you can ask.
+
+## Guidance on using databases / data for deployed apps
+
+This section of the guidance covers best practice for using data on the platform for deployed apps. If you want to know how to setup an app bucket go to the [deploy an R shiny app section](#deploying-an-r-shiny-app) of the guidance.
+
+### What to consider when giving an app access to data
+
+We do not give any app on the Analytical Platform access to our databases we expose and create for analysis. This is because apps can be accessed to users outside of the analytical platform, so we like to air gap the permissions and access of the App. For ease we are going to refer to two types of AP roles, the first is you the AP-user (most likely you) who has access to the tools on the Analytical Platform like R/Python, GitHub, etc. Then the other role is the App just like users each App will have individual permissions but will only ever be able to access things on the platform that are specifically for the App (so much more restricted access and also does not represent an individual). Whoever is using the deployed app is using it with that App's role. What this means is:
+
+- Apps cannot access (read/write) S3 buckets created by the AP users (instead they have their own buckets prefixed with `alpha-app-`)
+- Apps cannot access Databases created for the AP users (these are all of the curated databases created by the Data Engineers)
+- Users can access read/write to App buckets (permissions to do this are given via the Control Panel).
+
+So if you create an app that needs specific data from our databases then you need to consider the following practices:
+
+#### Data Minimisation
+
+When writing data to your app-bucket you should only ever give the app what it needs. I.e. even though your analysis might require personal or low level data. If the calculations on your App do not need the data at this lower level, then only write data to the app-bucket that is an aggregate of the data you take. You should never put data in the app-bucket that you do not want users to see (even if the code behind the app itself has no method to expose that data to the user via the app).
+
+
+#### Writing data to an App bucket
+
+There are multiple ways you can do this and it is up to you to determine how best to do it.
+
+- You could manually run a script from your AP User instance that writes the necessary data to the app bucket whenever it needs updating
+- You could write an airflow script that writes data from a database to the app bucket (this is because the airflow role will be seperate to the app role). Meaning the app still only has access to it's very specific area in AWS.
```

Comment:
You don't need to spell out that the Airflow role is different, it won't occur to many users. Also this should be one sentence not two.